src/test/java/core/basesyntax/dao/impl/StorageDaoImplTest.java

```diff
@@ -0,0 +1,94 @@
+package core.basesyntax.dao.impl;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.storage.Storage;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class StorageDaoImplTest {
+    private static final StorageDao storageDao = new StorageDaoImpl();
+    private static final int BANANA_VALID_QUANTITY = 152;
+    private static final int APPLE_VALID_QUANTITY = 90;
+    private static final int MANGO_VALID_QUANTITY = 90;
+    private static final int INVALID_QUANTITY = -1;
+    private static final String PRODUCT_BANANA = "banana";
+    private static final String PRODUCT_APPLE = "apple";
+    private static final String PRODUCT_MANGO = "mango";
+
+    @BeforeEach
+    void setUp() {
+        Storage.fruits.putAll(Map.of(
+                PRODUCT_BANANA, BANANA_VALID_QUANTITY,
+                PRODUCT_APPLE, APPLE_VALID_QUANTITY
+        ));
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.fruits.clear();
+    }
+
```

Comment:
I suggest to move cleanup logic to BeforeEach, to be sure that db is clear and test data in not affected by earlier changes

Reply:
Done, thanks 👍